ethereum/gnosis_protocol_v2/insert_trades.sql

```diff
@@ -0,0 +1,196 @@
+CREATE OR REPLACE FUNCTION gnosis_protocol_v2.insert_trades(start_ts timestamptz, end_ts timestamptz=now()) RETURNS integer
+    LANGUAGE plpgsql AS
+$function$
+DECLARE
+    r integer;
+BEGIN
+    WITH rows AS (
+        WITH trades_with_prices AS (
+            SELECT evt_block_time               as block_time,
+                   evt_tx_hash                  as tx_hash,
+                   owner,
+                   "orderUid"                   as order_uid,
+                   "sellToken"                  as sell_token,
+                   "buyToken"                   as buy_token,
+                   ("sellAmount" - "feeAmount") as sell_amount,
+                   "buyAmount"                  as buy_amount,
+                   "feeAmount"                  as fee_amount,
+                   s.price                      as sell_price,
+                   b.price                      as buy_price
+            FROM gnosis_protocol_v2."GPv2Settlement_evt_Trade" trades
+                     LEFT OUTER JOIN prices.usd as s
+                                     ON trades."sellToken" = s.contract_address
+                                         AND s.minute > TO_DATE('2021/03/03', 'YYYY/MM/DD') --! Deployment Date
```

Comment:
Do you have a specific reason for not using `start_ts` in the `ON` clause here (and also in similar statement below)?

Reply:
No specific reason, I could probably use the start ts here as well!

Reply:
Actually, perhaps this inequality is not needed at all since in the following line we have an equality condition on the very same field. I had introduced this a long time ago thinking that the performance might be better if we restrict first the date to be in a certain range and followed by an equality. Perhaps this was a silly thought and is completely unnecessary. WDYT?

Reply:
Makes sense to remove.

Reply:
Done!